ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -1703,6 +1816,12 @@ func createPVMInstance(d *schema.ResourceData, client *instance.IBMPIInstanceCli
 	if tags, ok := d.GetOk(Arg_UserTags); ok {
 		body.UserTags = flex.FlattenSet(tags.(*schema.Set))
 	}
+	if vsn, ok := d.GetOk(Arg_VirtualSerialNumber); ok {
+		vsnListType := vsn.([]interface{})
```

Comment:
Not sure why this is declared here and passed as a parameter to the func in next line?

Reply:
It needs to convert the list type (what terraform is taking) to a `CreateServerVirtualSerialNumber` model to send to SB